src/widgets/programs/programs-overview.component.tsx

```diff
@@ -35,7 +39,7 @@ export default function Programs(props: ProgramsOverviewProps) {
   return (
     <SummaryCard
       name={t("care programs", "Care Programs")}
-      link={`/patient/${patientUuid}/chart/programs`}
+      link={`${props.basePath}`}
```

Comment:
This should be programsPath not props.basePath

Reply:
Fixed